src/fsharp/FSharp.Core/seq.fs

```diff
@@ -1488,3 +1488,72 @@ namespace Microsoft.FSharp.Collections
                                 [|SR.GetString SR.inputMustBePositive; count|]
             mkDelayedSeq (fun () ->
                 source |> toArray |> Array.splitInto count :> seq<_>)
+
+        [<CompiledName("RemoveAt")>]
+        let removeAt (index: int) (source: 'T seq) : 'T seq =
```

Comment:
We need to consider the spec w.r.t. out-of-range indexes, and whether it differs from Array and List.  Here for overly-large indexes the out-of-range can't be discovered until the end of iteration of the list, and it would then seem wrong to raise an exception at the very end of iteration